### ISSUE TITLE
pip install fails 

### ISSUE THREAD

**Issue by tassiaquaresma** [PedestrianDynamics/jupedsim#1253]
I encouter a problem installing jupedsim.

These are the steps I follow:

```
conda create --name jps_env python=3.10 
```

```
conda activate jps_env
```

```
conda install pip
```

```
which pip
> /opt/anaconda3/envs/jps_env/bin/pip
```

```
pip install jupedsim
```

<img width="1223" alt="Screenshot 2023-10-25 at 14 15 04" src="https://github.com/PedestrianDynamics/jupedsim/assets/52387201/da527b13-3895-4e6f-84bc-5ebd26ca575f">

Thank you for your help!

**Comment by Ozaq** [comment]
Yeah we can reproduce the issue, looks like its related to installs from then source dist package. We do not (yet) provide wheels for intel Macs.

**Comment by Ozaq** [comment]
Fixed on `1.0.3` and `master`
